src/screens/Harbor/ProcessStake/ProcessStake.tsx

```diff
@@ -0,0 +1,227 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from 'react';
+import { RefreshControl, ScrollView, View } from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { useTranslation } from 'react-i18next';
+import { ethers } from 'ethers';
+import { formatEther, parseEther } from 'ethers/lib/utils';
+import { BottomSheetRef, Header } from '@components/composite';
+import { Spacer, Text } from '@components/base';
+import { COLORS } from '@constants/colors';
+import { scale } from '@utils/scaling';
+import { HarborTitle, RateInfo, StakedBalanceInfo } from './components';
+import { styles } from './styles';
+import { useWalletStore } from '@entities/wallet';
+import { useBalanceOfAddress, useKeyboardHeight } from '@hooks';
+import { InputWithoutTokenSelect } from '@components/templates';
+import { ExplorerAccount, Token } from '@models';
+import { AMB_DECIMALS } from '@constants/variables';
+import { CryptoCurrencyCode } from '@appTypes';
+import { TokenUtils } from '@utils/token';
+import { PrimaryButton } from '@components/modular';
+import { useHarborStore } from '@entities/harbor/model/harbor-store';
+import { NumberUtils } from '@utils/number';
+import { DEFAULT_STAKE_PREVIEW } from '@entities/harbor/constants';
+import { BottomSheetHarborPreView } from '@features/harbor/components/harbor-preview';
+
+export const ProcessStake = () => {
+  const [previewData, setPreviewData] = useState(DEFAULT_STAKE_PREVIEW);
+
+  const { t } = useTranslation();
+  const bottomSheetRef = useRef<BottomSheetRef>(null);
+  const [amountToStake, setAmountToStake] = useState('');
+  const [inputError, setInputError] = useState('');
+
+  const { data: harborData, updateAll, loading } = useHarborStore();
+  const { wallet } = useWalletStore();
+
+  const { apr: harborAPR, stakeLimit, userStaked } = harborData;
+
+  const {
+    data: selectedAccountBalance,
+    refetch: refetchAmbBalance,
+    loading: accountDataLoading
+  } = useBalanceOfAddress(wallet?.address || '');
+
+  const refetchAll = async () => {
+    try {
+      if (wallet?.address) {
+        updateAll(wallet.address);
+      }
+      if (refetchAmbBalance) {
+        refetchAmbBalance();
+      }
+    } catch (error) {
+      // ignore
```

Comment:
do not ignore catch block, always throw error. It's the reason why we use try {} catch...